src/main/msp/msp_serial.h

```diff
@@ -39,23 +38,24 @@ typedef enum {
 } mspEvaluateNonMspData_e;
 
 #define MSP_PORT_INBUF_SIZE 64
```

Comment:
MSP does not need to vaid for response in most cases - just queue all requests.
Adding numbering to packet header will help (and allow stopping processing is sequence number does not match).
One idea behind memory-based MSP rewrite was to separate command processing from transfer mechanism. One message in, at most one message reply. All headers and checksums are handles by transport layer, `mspFcProcessOutCommand` does not need to know any details.
And `mspSerialEncode` does not care about packet structure (except for error codes)